decidim-proposals/spec/models/decidim/proposals/proposal_spec.rb

```diff
@@ -11,6 +11,7 @@ module Proposals
       include_examples "has feature"
       include_examples "has scope"
       include_examples "has category"
+      include_examples "has reference"
```

Comment:
I'd create `feature` examples that check the reference is actually shown somewhere in the UI.